services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return get_object_or_404(Movie, pk=movie_id)
```

Comment:
Instead of `get_object_or_404 `use `Movie.objects.get(id=movie_id)`
Correct this in other places